### PR TITLE
Prep for 4.1.6 release

### PR DESCRIPTION
**Pull request by MGatner**
**Description**
Updates changelog and version references for `4.1.6`.

**Checklist:**
- [X] Securely signed commits
- [X] Component(s) with PHPDoc blocks, only if necessary or adds value
- [X] Unit testing, with >80% coverage
- [X] User guide updated
- [X] Conforms to style guide

### CODE REVIEW

File: admin/RELEASE.md [hunk 1]
```diff
@@ -1,31 +1,30 @@
...
 - **bug** ... PRs that fix bugs
 - **enhancement** ... PRs to improve existing functionalities
 - **new feature** ... PRs for new features
 - **refactor** ... PRs to refactor
 
-And PRs that have the breaking changes must have the following label:
+PRs with breaking changes must have the following additional label:
 - **breaking change** ... PRs that may break existing functionalities
 
+To auto-generate, start drafting a new Release and use the "Auto-generate release notes" button.
```
Comment (kenjis) [inline admin/RELEASE.md:20]: How do we do? It seems it needs a valid tag. When do we tag?
Screenshot 2021-12-25 9 19 19

Comment (MGatner) [reply admin/RELEASE.md:20]: At this stage you won't actually complete the release, so you can type anything into the tag dropdown.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,65 @@
...
+* fix: VENDORPATH definition by @kenjis in https://github.com/codeigniter4/CodeIgniter4/pull/5453
+* fix: Throttler does not show correct token time by @kenjis in https://github.com/codeigniter4/CodeIgniter4/pull/5470
+* fix: directory_mirror() throws an error if destination directory exists by @kenjis in https://github.com/codeigniter4/CodeIgniter4/pull/5493
+
+**New Features**
+
+* feat: add filter to check invalid chars in user input by @kenjis in https://github.com/codeigniter4/CodeIgniter4/pull/5227
+
+**Enhancements**
+
```
Comment (kenjis) [inline CHANGELOG.md:39]:
```suggestion

* Add support for PHP 8.1 by @paulbalandan in https://github.com/codeigniter4/CodeIgniter4/pull/4883
```

Comment (MGatner) [reply CHANGELOG.md:39]: Good catch!

**Comment by TimZ99** [comment]
Looks like it's ready to merge/ release.

**Comment by cijagani** [comment]
🎉🎉

**Pull request merged**